src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformAllelicSegmentation.java

```diff
@@ -0,0 +1,97 @@
+package org.broadinstitute.hellbender.tools.exome.segmentation;
+
+import org.apache.commons.io.FilenameUtils;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.tools.exome.ModeledSegment;
+import org.broadinstitute.hellbender.tools.exome.SegmentUtils;
+import org.broadinstitute.hellbender.tools.exome.allelefraction.AllelicPanelOfNormals;
+import org.broadinstitute.hellbender.tools.exome.alleliccount.AllelicCountCollection;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Created by davidben on 5/23/16.
+ */
+@CommandLineProgramProperties(
+        summary = "Segment genomic data into regions of constant minor allele fraction.  Only supports one sample input.",
+        oneLineSummary = "Segment genomic data into regions of constant minor allele fraction",
+        programGroup = CopyNumberProgramGroup.class
+)
+public class PerformAllelicSegmentation extends CommandLineProgram {
```

Comment:
also, perhaps `PerformAlleleFractionSegmentation` to be consistent with other class names, but i'm fine with either